lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -37,83 +37,42 @@ def __init__(self, subsection, course):
         self.all_total = None  # aggregated grade for all problems, regardless of whether they are graded
         self.locations_to_weighted_scores = OrderedDict()  # dict of problem locations to (Score, weight) tuples
 
-    @lazy
+        self._scores = None
+
+    @property
     def scores(self):
         """
         List of all problem scores in the subsection.
         """
-        log.info(u"Persistent Grades: calculated scores property for subsection {0}".format(self.location))
```

Comment:
yes.  I changed `score` so it is no longer a lazy attribute.  Generally speaking, the `invalidate` method is not really recommended to be used on lazy attributes.  If we do end up needing to use it, it generally implies that we should re-think our usage of it.  

See the sample log output below to determine whether this is still really needed.

I *did* keep the log statements for all the lazy attributes in the `CourseGrade` class.